doc/release/index.rst

```diff
@@ -61,6 +61,12 @@ For information about earlier versions, see :doc:`eol_versions`.
             -   End of support
             -   Versions
 
+        *   -   :doc:`3.2 </release/3.2.0>`
+            -   **August 21, 2024**
+            -   **August 21, 2026**
+            -   **Not planned yet**
+            -   | :tarantool-release:`3.2.0`
+
         *   -   :doc:`3.1 </release/3.1.0>`
             -   **April 16, 2024**
             -   **April 16, 2026**
```

Comment:
We have no `3.0`, `3.1` and `3.2` as separate objects of support. In fact, `3.0` is already EOL (not sure about EOS) -- the development is now on 3.2 with backporting bugfixes to 3.1. We need to elaborate a good way to present the real commitments.

It is likely not a goal of this PR, so just noted.

Reply:
Good point, thanks. Will discuss with PM.